hpc/LoadBalancer.cpp

```diff
@@ -25,15 +25,19 @@ void clear_url(std::string directory) {
 }
 
 void launch_hq_with_alloc_queue() {
-    std::system("hq server stop &> /dev/null");
+    std::system("./hq server stop &> /dev/null");
 
-    std::system("hq server start &");
+    std::system("./hq server start &");
     sleep(1); // Workaround: give the HQ server enough time to start.
 
     // Create HQ allocation queue
     std::system("hq_scripts/allocation_queue.sh");
 }
 
+bool file_exists(const std::string& path) {
```

Comment:
kind of redundant?

Reply:
ah true - habit, thought I would use it more